### ISSUE TITLE
XBox 360 Headset, Windows 10, No Audio

### ISSUE THREAD

**Issue by JSeddon** [mumble-voip/mumble#1782]
I recently upgraded to Windows 10, and have found that my XBox 360 wired headset, which functioned properly under Windows 7, no longer plays audio through the earpiece.  

I can sometimes, but not always, use the microphone and get audio through the computer speakers.  I have reinstalled and confirmed the drivers are showing correctly in Device Manager.  I have confirmed that the headset still functions correctly in Skype.  And I have confirmed that I am able to use a different USB headset in Mumble with Windows 10.

**Comment by Kissaki** [comment]
Does that headset use its own sound card?
Did you try increasing the jitter buffer for output?

The output never works,
but the mic sometimes works?

**Comment by JSeddon** [comment]
The XBox 360 controller is USB, and the headset plugs into the controller.

 

I did not try adjusting the jitter buffer.

 

I was unable to get audio through the headset in Mumble only; I could get Mumble audio through my computer speakers, and the headset speakers worked in Windows itself, and in other communication programs.  The mic appeared to function correctly at all times.

 

In the meantime, I have switched to a different USB headset, which is functioning without any issues, and I have done no further troubleshooting.
